projects/ui/src/lib/components/po-field/po-checkbox/samples/sample-po-checkbox-labs/sample-po-checkbox-labs.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+
+import { PoRadioGroupOption } from '@portinari/portinari-ui';
+
+@Component({
+  selector: 'sample-po-checkbox-labs',
+  templateUrl: './sample-po-checkbox-labs.component.html'
+})
+export class SamplePoCheckboxLabsComponent implements OnInit {
+
+  checkbox: boolean | object;
```

Comment:
alterar para `boolean | null` por que desta forma eu consigo mandar objetos, por exemplo:

`this.checkbox = { test: 'test' };`

Reply:
ok